Common/Camera.cs

```diff
@@ -27,23 +29,19 @@ public static void Move(Vector3 move)
             position += move;
         }
 
-        public static void Rotate(Vector4 rotate)
+        public static void Rotate(Vector3 rotate)
         {
             //To rotate a quaternion we can just multiply it with another quaternion
             //We generate the quaternion from an axis angle which is an axis defined by the x, y and z components and
             //an angle defined by the w component of the vector
-            rotation *= Quaternion.FromAxisAngle(rotate.Xyz, rotate.W);
+            rotation += rotate;
         }
         
-        public static void Use(Shader shader)
+        public static Matrix4 GetView()
```

Comment:
Just for consistency, this should probably be a get-only property, similar to the projection matrix.